make/riot/riot-lfc.mk

```diff
@@ -0,0 +1,26 @@
+RIOT_MK_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
+
+# Check if required environment variables exist
+ifndef LF_MAIN
+  $(error LF_MAIN is not defined. Please define it!)
+endif
+
+# Name of your RIOT application
+APPLICATION ?= $(LF_MAIN)
+
+# Path of generated lf c-code
+LF_SRC_GEN_PATH ?= $(CURDIR)/src-gen/$(LF_MAIN)
```

Comment:
An unfortunate detail about lfc is that where the generated-code is put is a little complicated:
1. `~/src/Hello.lf` -> `~/src-gen/Hello`
2. `~/Hello.lf` -> `~/src-gen/Hello`
3. `~/src/lib/Hello.lf` -> `~/src-gen/lib/Hello

I think we should ignore (2) where it is not in a `src` directory, for our template we say that the LF files must reside under a `src` directory. But for (3) to work, LF_MAIN must be set to `lib/Hello`, which might actually be reasonable

Reply:
Aaahh that is good to know, I will think about a good solution